osutil/disks/disks.go

```diff
@@ -379,7 +389,8 @@ func (d *disk) FindMatchingPartitionUUID(label string) (string, error) {
 		return partuuid, nil
 	}
 
-	return "", fmt.Errorf("couldn't find label %q", label)
+	fmt := "could not find label %q: %w"
+	return "", xerrors.Errorf(fmt, label, ErrFilesystemLabelNotFound)
```

Comment:
this works but is not immediately clear to me why this instead  of 

```
type FilesystemLabelNotFoundError {
    Label string
}
```

the new error produced here is ok, but not great, with some word repetition in it

Reply:
sure I can use that instead